client/my-sites/importer/style.scss

```diff
@@ -18,7 +18,7 @@
 	margin-bottom: 16px;
 }
 
-.importer__section-description {
+.site-settings__importer-section-description {
```

Comment:
Little nit here, it feels odd to have the `site-settings__importer-section-description` class under `/importer/`, we're sort of defining usage in one directory and styles in another ( `importer` vs `site-settings`).

I feel like this sort of points out something else, that maybe the import section shouldn't live in `site-settings` 🤷‍♂️ 

Let's ignore for now but if we end up working on a new importer section we should move it somewhere that makes a bit more sense.